README.md

```diff
@@ -30,6 +30,23 @@ The gradebook builds on the awesome work of @jwo and @matthiasak. Each student y
 
 Data is cached in localStorage so as not to DDoS TIYO (too much). Clicking the "Refresh Grades" button will scrape the site once again and store the new data.
 
+### Homework
+
+The Homework module builds upon the inspiration from @jah2488.  It allows communication to an authorized ruby process running on your computer to rapidly clone and setup git based projects on your computer.  This allows for easy cloning of PR's, Git Repos, Gists, and various other git related links.  For full documentation please read more about the underlying tool newline_hw [here](https://github.com/TIYDC/newline_hw#newlinehw).
+
```

Comment:
Alright, so leave the install instructions, on drop down when no heartbeat.   Show use and upgrade instructions on dropdown when heartbeat detected?

Also add some indicator that they should click back to see if it's installed?